sdv/constraints/tabular.py

```diff
@@ -195,11 +197,11 @@ class GreaterThan(Constraint):
     will be added to the diff to reconstruct the ``high`` column.
 
     Args:
-        low (str or int):
-            Either the name of the column that contains the low value,
+        low (str or list[str] or int):
```

Comment:
I think we should clean this up. We should either:
- Be consistent and mention `int` and `str` everywhere (so, make it `list[str or int]`, and see if there are other places where we should add `int` too)
- Drop `int` and mention `str` only (since `int` is just one of the many other types supported)
- Change everything to `object`, since anything that is hashable can be a column name.

I personally would go for the second option for reading simplicity, since we mention `str` only in many other palaces. Advanced users will know that other things also work anyway.

Reply:
Wouldn't that be a bit awkward for the user to understand?
- option 1 is confusing in a sense that `list[int]` is problematic and would not work
- option 2 is the least confusing; but it also hides that `int` is supported
- option 3 is very general imo

I guess then make the change to `str or list[str]`? or would it just be `str`?

Reply:
> * option 1 is confusing in a sense that `list[int]` is problematic and would not work

Why would it not work? Here the `int` is just a `column_name`, so if you have multiple column names that are integer values, you can pass a list of integers.

> * option 2 is the least confusing; but it also hides that `int` is supported

Right, but `int` is not the only type that is being silently supported, so I think it is fine. 

> I guess then make the change to `str or list[str]`? or would it just be `str`?

I would say `str or list[str]`.

Reply:
> Why would it not work? Here the `int` is just a `column_name`, so if you have multiple column names that are integer values, you can pass a list of integers.

Right, I got what you mean now!